### PR TITLE
feature/12

### PR DESCRIPTION
**Pull request by vladkvts**
Created service for getting 10 latest post from user's instagram.
To test service, instantiate it and call the GetLatestPost(username) method, where username is one of the following: intelli_test, vlod_ahotnek, yakovets_vlad . As i have to add user to the sandbox first, i had to use my own instagram ( and create a few too ))
Also changed IInstagramService interface (GetLatestPosts() changed for  GetLatestPosts(string username)  
 to get posts from certain user ).

### CODE REVIEW

**Comment by vladkvts** [comment]
Classes in the region "ClassesForDeserialization" and their fields named that way to get data from JSON response and to create objects from it.

File: InstaBotPrototype/InstaBotPrototype/Services/Instagram/InstagramService.cs [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+            string response = GetResponse(getRecentMedia);
+            Post posts = JsonConvert.DeserializeObject<Post>(response);
+            return posts.data;
+
+        }
+
+        public  IEnumerable<string> GetLatestPosts(string username)
+        {
+            string userId = GetUserId(username);
+            List<ImageData> posts = new List<ImageData>();
```
Comment (AkulovBogdan) [inline InstaBotPrototype/InstaBotPrototype/Services/Instagram/InstagramService.cs:50]: Возможно тут лучше var, поскольку в правой части очевидно с чем мы имеем дело

File: InstaBotPrototype/InstaBotPrototype/Services/Instagram/InstagramService.cs [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            UsersInfo followers = GetFollowers(userId);
+            foreach (var user in followers.data)
+            {
+                posts.AddRange(GetRecentUserPosts(user.id));
+            }
+            posts.Sort(new ImageComparer());
+            List<string> latestPosts = new List<string>();
```
Comment (AkulovBogdan) [inline InstaBotPrototype/InstaBotPrototype/Services/Instagram/InstagramService.cs:57]: Тут тоже

File: InstaBotPrototype/InstaBotPrototype/Services/Instagram/InstagramService.cs (hunk 1 above)
Comment (demosglok) [reply InstaBotPrototype/InstaBotPrototype/Services/Instagram/InstagramService.cs:50]: резонное замечание, но пока не критичное, смержу как есть, ибо конец спринта

**Pull request merged**